sdk/openai/Azure.AI.OpenAI/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.14 (Unreleased)
+## 1.0.0-beta.15 (Unreleased)
```

Comment:
We can omit the "vNext" entry -- the last version was actually a sequencing error on my part and I believe we generally want the released release notes to just start with the current (released) version.

Reply:
Make sense. Will remove it.